draft-ietf-oauth-resource-metadata.xml

```diff
@@ -206,11 +206,9 @@
 	    <vspace/>
 	    OPTIONAL.
 	    URL of the protected resource's JWK Set <xref target="JWK"/> document.
-	    This contains public keys belonging to the protected resource.
-	    For instance, this JWK Set MAY contain encryption key(s)
-	    that are used to encrypt access tokens to the protected resource.
-	    The JWK Set MAY also contain signing key(s) that the resource
-	    server uses to sign responses.
+	    This contains public keys belonging to the protected resource, such as
+	    signing key(s) that the resource server uses to sign resource responses.
+	    This URL MUST use the <spanx style="verb">https</spanx> scheme.
 	    When both signing and encryption keys are made available,
 	    a <spanx style="verb">use</spanx> (public key use) parameter
 	    value is REQUIRED for all keys in the referenced JWK Set
```

Comment:
@aaronpk and @selfissued, Is the intent here that if no key usage is specified, the keys are signing keys? Why not always be explicit. Either explicitly state that all keys are signing keys by default, and if encryption keys are used, key usage should be added, or alternatively make a key usage parameter mandatory, whether the keys are signing keys only, or if there are a mix of signing and encryption keys.

Reply:
This is exactly the same language as in the `jwks_uri` definition at https://www.rfc-editor.org/rfc/rfc8414.html#section-2.  I would rather keep the OAuth metadata specs consistent in their treatment of this parameter.

Are you OK approving this on that basis, @PieterKas ?  Thanks.

Reply:
@selfissued thanks for pointing that out. 

I was thinking through whether we will break something if we move away from the implicit key usage defined in the Authorization Server Metadata spec and concluded that we probably end up making the developers life more difficult as they now have two different models for validating resource metadata, creating opportunity for confusion and reducing re-use. So assuming they are parsing and validating authorization server metadata correctly, they should be able to do the same for resource metadata. So, I think with that, I am good with approving the changes.